dirty_cat/super_vectorizer.py

```diff
@@ -26,7 +28,11 @@ def _has_missing_values(df: Union[pd.DataFrame, pd.Series]) -> bool:
     return any(df.isnull())
 
 
-def _replace_false_missing(ser: pd.Series) -> pd.Series:
```

Comment:
Please add a small docstring to describe to the developer what this function does.
It does not need to be a properly formatted docstring describing all parameters, as this is an internal function, we just need to describe why this function